openapi-gui.js

```diff
@@ -1,15 +1,19 @@
-#!/usr/bin/env node
+#!_usr/bin/env node
 
 'use strict';
 
 const util = require('util');
 
 const express = require('express');
 const compression = require('compression');
+const widdershins = require('widdershins');
+const shins = require('shins');
 
 const ourVersion = require('./package.json').version;
 var definition = {openapi:"3.0.0",info:{title:"API",version:"1.0.0"}};
 
+process.on('unhandledRejection', r => console.log(r));
```

Comment:
Maybe a comment as to why this is necessary?

Reply:
This looks like a leftover from a previous hack session.

Reply:
Actually, this is super useful -- it causes the runtime to render decent stack traces on errors.

Reply:
Ooh. Keep it then, stacktraces from Promises are usually hateful.

Reply:
Added a comment.